tests/wiseagents/agents/test_challenger.py

```diff
@@ -65,18 +69,26 @@ def get_connection_string():
 
 
 def response_delivered(message: WiseAgentMessage):
+    global assertError
     with cond:
         response = message.message
-        assert "4 medals" not in response
+        try:
+            assert "4 medals" not in response
+        except AssertionError:
+            assertError = AssertionError
         print(f"C Response delivered: {response}")
         cond.notify()
 
 
 @pytest.mark.needsllama
 def test_cove_challenger():
+    global assertError
+    groq_api_key = os.getenv("GROQ_API_KEY")
```

Comment:
Since the tests have been updated to use Groq, we also need to update the tests to remove the annotations that currently result in them being skipped in CI (e.g., `@pytest.mark.needsllama`, `@pytest.mark.needsllm`, should be removed in the updated tests).

Reply:
Marking this as resolved since the tests will be updated via https://github.com/wise-agents/wise-agents/issues/195